cmd/commands.go

```diff
@@ -305,6 +305,50 @@ the KEY=VALUE format will be loaded into the Caddy process.
 		},
 	})
 
+  RegisterCommand(Command{
+		Name:  "storage-export",
```

Comment:
:thinking: yeah either one is fine. I thought it would be fun to set up a subcommand. I might take a look at it in the coming days, I'm curious to see how it would work.

Reply:
Pushed a commit, got it working :+1: was pretty simple.

Also as an aside, please make sure your editor is configured to use `.editorconfig`; your code was using spaces instead of tabs among other minor style issues. Use `gofmt -w -s <path>` to apply fixes.